LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2019, IBM
```

Comment:
Copyright 2019 IBM

Reply:
Is it not supposed to be IBM Corporation?